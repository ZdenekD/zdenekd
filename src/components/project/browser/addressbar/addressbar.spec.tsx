import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../../../__test__/utils/helpers';
import Addressbar from '.';

describe('Addressbar', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Addressbar url="http://url" />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-addressbar');

        expect(component.exists()).toBe(true);
    });
});
