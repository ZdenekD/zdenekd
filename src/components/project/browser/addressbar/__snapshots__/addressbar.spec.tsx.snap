// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Addressbar match snapshot 1`] = `
<div class="addressbar http"
     data-test="component-addressbar"
>
  <a href="http://url"
     target="_blank"
     rel="noreferrer noopener"
     aria-label="Otevřít referenci v novém okně"
  >
    http://url
  </a>
</div>
`;
